CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+## Contributing to the Plugin
+
+### Set up your environment
+
+To discover plugins, Grafana scans a plugin directory, the location of which depends on your operating system.
+
+   1. Create a directory called `grafana-plugins` in your preferred workspace.
+
+   2. Find the plugins property in the Grafana configuration file and set the plugins property to the path of your grafana-plugins directory. Refer to the [Grafana configuration documentation](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#plugins) for more information.
+
+      ```
+        [paths]
+        plugins = "/path/to/grafana-plugins"
+      ```
+
+   3. Restart Grafana if it’s already running, to load the new configuration.
+
+#### Using Docker
+  1. To set up Grafana for plugin development using Docker, run the following command:
+      ```
+      docker run -d -p 3000:3000 -v "$(pwd)"/grafana-plugins:/var/lib/grafana/plugins --name=grafana grafana/grafana:7.0.0
```

Comment:
I see that this is probably a generated file, but are you sure it is best to point people to the 7.0.0 version?

Reply:
Great idea! Thanks!